### ISSUE TITLE
period explanation

### ISSUE THREAD

**Issue by mmjo** [chubin/wttr.in#758]
I could not find an explanation for the period parameter.
Is the server using the value of period as a timeout for caching?

**Comment by chubin** [comment]
No. `period` in multi-location one-line queries like this:

```
wttr.in/London:Stockholm:Moscow\?format\="%%l:+%%c%%20%%t%%60%%w&period=60
```

is used to determine what location has to be displayed for the current query.
The service has to cycle through all locations, and return the first one, then the second one, etc.

But how does it know what location should be displayed for some particular query?
If it knows the time interval between the queries, it can guess what location should be displayed now.

Basically, it is not the interval between the queries, but rather how often the locations
should be changed. So even if you make a query each second, but you have `period=60`,
the location will be changed once a minute.

**Comment by mmjo** [comment]
Thanks for the explanation
